sphinx_gherkindoc/writer.py

```diff
@@ -244,6 +245,8 @@ def ticket_url_or_tag(tag: str) -> str:
         return _value_with_url(tag, url) if url else tag
 
     def tags(tags: List[str], *parent_objs: behave.model_core.BasicStatement) -> None:
+        # this appends the tags to the previous set
+        tag_set.update(set(tags))
```

Comment:
The `set.update` method can take any iterable, so you don't need to convert tags to a set first.